### ISSUE TITLE
error in ProbOhemCrossEntropy2d

### ISSUE THREAD

**Issue by Serge-weihao** [Reagan1311/DABNet#5]
I got this error in ProbOhemCrossEntropy2d, when trainning on cityscapes.
![屏幕截图(2)_LI](https://user-images.githubusercontent.com/40890323/63115983-1ac69900-bfcb-11e9-88b3-0e39339cb350.jpg)

**Comment by Reagan1311** [comment]
May I ask what code did you change? It seems that you didn't set the ignore label in the loss function.

**Comment by Serge-weihao** [comment]
> 
> 
> May I ask what code did you change? It seems that you didn't set the ignore label in the loss function.

ignore label is setted in your train.py, which is 255, and the problem seems to be caused by tensor type.

**Comment by Serge-weihao** [comment]
I replace 1-valid_mask with ~valid_mask, and the code for trainning cityscapes can run now.

**Comment by Serge-weihao** [comment]
![图片](https://user-images.githubusercontent.com/40890323/63223257-81270380-c1e5-11e9-9f76-2b016e87a822.png)
And I want to know the mIoU of 69.57​% is from val set or test set?

**Comment by Reagan1311** [comment]
This result were trained on the training set and tested on the val set.
